paas_app_charmer/observability.py

```diff
@@ -47,13 +48,22 @@ def __init__(  # pylint: disable=too-many-arguments
             jobs=jobs,
             relation_name="metrics-endpoint",
         )
-        self._logging = LogProxyConsumer(
-            charm,
-            alert_rules_path=os.path.join(cos_dir, "loki_alert_rules"),
-            container_name=container_name,
-            log_files=[str(log_file) for log_file in log_files],
-            relation_name="logging",
-        )
+        if enable_pebble_log_forwarding():
+            import charms.loki_k8s.v1.loki_push_api  # pylint: disable=import-outside-toplevel
```

Comment:
Perhaps document the reason this needs to be disabled please as a comment?

Reply:
Added, thanks!

Reply:
Should we include the reason we're importing here? I.e., because we need to support either which is why we can't place it at the top level?

Reply:
Yes, added, thanks